WcaOnRails/app/views/competitions/_registration_requirements.html.erb

```diff
@@ -119,13 +119,18 @@
   <br>
 <% else %>
   <% if @competition.all_guests_allowed? %>
-    <%= t('competitions.competition_info.guests_free.anyone') %>
+    <%= t('competitions.competition_info.guests_free.free') %>
```

Comment:
The "free free" in `guests_free.free` is a little awkward. I can change to something else if you have a good suggestion @gregorbg (trying to keep the number of line changes as low as possible for now).

Reply:
It's only a translation string index/key. No need to make this matter even more complicated :)